calm_adapter/calm_adapter/src/main/scala/uk/ac/wellcome/calm_adapter/CalmXmlResponse.scala

```diff
@@ -80,6 +80,10 @@ case class CalmSummaryResponse(val root: Elem, retrievedAt: Instant)
 
   val responseTag = "SummaryHeaderResponse"
 
+  val suppressedFields = List(
+    "UserWrapped7"
```

Comment:
Would be good to have a comment here explaining why we're suppressing this particular field. How would we know if we didn't have to suppress this field any more? Or if we had to suppress new fields?